terraform_alert_config/import_adapter/README.md

```diff
@@ -0,0 +1,68 @@
+# Prepare workspace
+## gcloud
+Information about installation [gcloud installation](https://cloud.google.com/sdk/docs/install)
+## Terraform
+Information about installation [Terraform installation](https://www.terraform.io/downloads.html)
+## Kubernetes
+Information about installation [Kubernetes installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+# Description
+Terraform configuration for deploying alerting on DICOM Adapter health. Only the import adapter is in scope initially.
```

Comment:
phrasing edit: 
"Terraform configuration for deploying alerting on DICOM Adapter health. Only the import adapter is in scope initially."
->
"Terraform configuration for deploying alerting on DICOM Import Adapter health. "